manager.py

```diff
@@ -102,6 +107,26 @@ def __init__(self,
             verbosity=self.v,
             logfile=self.logfile)
 
+        self.datalog = datalog
+        self.datalogflag = datalogflag
+        
+        self.a_flag()
+        self.d_flag()
+        self.make_data_log(self.datalog)
```

Comment:
Add short docstrings on a_flag() and d_flag() so it's more obvious what they are checking.